libs/designsystem/src/lib/components/chart/chart-js/chart-js.service.ts

```diff
@@ -163,36 +203,95 @@ export class ChartJSService {
   }
 
   private createOptionsObject(args: {
-    type: ChartType;
     customOptions?: ChartOptions;
     annotations?: AnnotationOptions[];
+    chartPeriod?: ChartPeriod;
+    dataLabelOptions?: ChartDataLabelOptions;
   }): ChartOptions {
-    const { type, customOptions, annotations } = args;
+    const {
+      customOptions,
+      annotations,
+      chartPeriod,
+      dataLabelOptions: chartDataLabelOptions,
+    } = args;
 
-    const typeConfig = this.chartConfigService.getTypeConfig(type);
+    const typeConfig = this.chartConfigService.getTypeConfig(this.chartType);
     const typeConfigOptions = typeConfig?.options;
     const annotationPluginOptions = annotations
       ? this.createAnnotationPluginOptionsObject(annotations)
       : {};
+
+    let defaultOptions: ChartOptions = {
+      locale: this.locale,
+      plugins: { tooltip: { callbacks: {} } },
+      scales: { y: { ticks: {} } },
+    };
```

Comment:
Would it make sense to move these to the CHART_GLOBAL_DEFAULTS object in `libs/designsystem/src/lib/components/chart/configs/global-defaults.config.ts `?

Reply:
I'm not 100% sure the result would be the same. I'm just triggered to ask as it seems to be applied to everything and will be overwritten by the type config and custom options.

Reply:
I cleaned it up a bit.